src/main/java/com/adobe/commerce/cif/api/CartApi.java

```diff
@@ -208,8 +186,8 @@ Cart postBillingAddress(
         @ApiParam(value = "The ID of the cart for the billing address", required = true)
         @PathParam("id") String id,
 
-        @ApiParam(value = "The billing address and an optional customer id that must be set if the cart belongs to a customer.", required = true)
-        AddressWrapper addressWrapper);
```

Comment:
Since I removed the customerId from the address wrapper only the address remained there. I removed `AddressWrapper` and directly used `Address` instead.